src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/DocumentConversionService.java

```diff
@@ -85,6 +92,41 @@ public byte[] flattenPdfDocument(byte[] document) {
         return document;
     }
 
+    private void flattenAnnotations(PDDocument doc) throws IOException {
+        for (PDPage page : doc.getPages()) {
+            List<PDAnnotation> annotations = page.getAnnotations();
+            for (PDAnnotation annotation : annotations) {
+                PDAppearanceStream appearanceStream = annotation.getNormalAppearanceStream();
+                if (appearanceStream != null) {
+                    transformAnnotationsToContentStream(doc, page, annotation, appearanceStream);
+                }
+            }
+            annotations.clear();
+        }
+    }
```

Comment:
Do you you feel is readable, but consider removing the the annotations var declaration. Simpler and reduces the need to clear.

`        for (PDPage page : doc.getPages()) {
            for (PDAnnotation annotation : page.getAnnotations()) {
                PDAppearanceStream appearanceStream = annotation.getNormalAppearanceStream();
                if (appearanceStream != null) {
                    transformAnnotationsToContentStream(doc, page, annotation, appearanceStream);
                }
            }
        }`

Looked at the stream equivalent, but not convinced it gains us anything.